### ISSUE TITLE
Somebody screwed up this page on the wiki

### ISSUE THREAD

**Issue by lucbettaieb** [Shopify/liquid#638]
https://github.com/Shopify/liquid/wiki/Liquid-for-Designers

**Comment by lucbettaieb** [comment]
Thought it was fixed - but sometimes it shows this page:
<img width="1023" alt="screenshot 2015-08-04 00 21 23" src="https://cloud.githubusercontent.com/assets/1468145/9055316/c80fef24-3a3e-11e5-82c3-098b183cdf3f.png">

While other times, it shows this page:
<img width="1037" alt="screenshot 2015-08-04 00 21 59" src="https://cloud.githubusercontent.com/assets/1468145/9055323/db406c36-3a3e-11e5-843c-abd01fd971fc.png">

**Comment by pushrax** [comment]
It's fixed now with commit 8bb68b4061d04bba3ee2b790fd41d0db9018a122 in the wiki. The issue was that a file was created with some trailing unicode chars which seemed to trip up GitHub. Original commit:

```diff
commit 40e02dfcaad8bcd7dcfe5d88ca6410686b241fdc
Author: Russell Miller <russell@adamm.net>
Date:   Mon Jul 27 08:01:07 2015 -0700

    Created Liquid for Designers  (markdown)

diff --git "a/Liquid-for-Designers\302\240.md" "b/Liquid-for-Designers\302\240.md"
new file mode 100644
index 0000000..ac864d5
--- /dev/null
+++ "b/Liquid-for-Designers\302\240.md"
@@ -0,0 +1 @@
+## # ### ````
```

Thanks to @dapi for removing it. @russellmiller3 was it you that created that file? May I ask why?

**Comment by russellmiller3** [comment]
Sorry, thought it was a test wiki
ᐧ
